monasca-log-api/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:2-alpine
```

Comment:
I think installing python via alpine's package manager results in a (slightly) smaller image than python:alpine but probably not enough to worry about. I would say we should use `python:2-alpine3.6` though, that base tag uses alpine:3.4.